src/ImageGallery.jsx

```diff
@@ -582,11 +582,13 @@ export default class ImageGallery extends React.Component {
 
             onTouchStart={event => this.slideToIndex.call(this, index, event)}
             onClick={event => this.slideToIndex.call(this, index, event)}>
-
-            <img
-              src={item.thumbnail}
-              alt={item.thumbnailAlt}
-              onError={onThumbnailError.bind(this)}/>
+              <img
+                src={item.thumbnail}
+                alt={item.thumbnailAlt}
+                onError={onThumbnailError.bind(this)}/>
+              <div >
```

Comment:
Add a class name called `image-gallery-thumbnail-label` so we can style this specifically